src/main/java/org/influxdb/impl/BatchProcessor.java

```diff
@@ -196,15 +216,16 @@ public static Builder builder(final InfluxDB influxDB) {
     return new Builder(influxDB);
   }
 
-  private BatchProcessor(final InfluxDBImpl influxDB, final ThreadFactory threadFactory, final int actions,
```

Comment:
Why is the weaker signature needed ?

Reply:
This is because I have been using more strict rules for check style at first by mistake. Sun style wants it like this. I don't think there is a real difference the only thing I can imagine it may affect javadoc at certain settings.

Reply:
OK